### PR TITLE
[Feature] Move Project - 1072

### LINKED ISSUE #1072: Move a project to a different portfolio
As a GovReady user,
I want to be able to move a project to a different portfolio,
so I can better manage my workspace.

It is important to be able to move projects to different portfolios to manage workspace and to be able to remove all projects from portfolio to delete the portfolio.

Related to #1071.

### PR DESCRIPTION
**Pull request by Yodart**
# Pull Request

## Description

Added the ability for a GovReady user to remove a project to a different portfolio

Closes #1072 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Unit Test - Tests moving a project to another portfolio

Scripts have been implemented to `controls/tests.py` to run it: `python manage.py test controls.tests `

## Checklist:

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by phani816** [comment]
[changes-files.zip](https://github.com/GovReady/govready-q/files/5697675/changes-files.zip)
Added functionality of edit and delete for portfolio

File: controls/tests.py [hunk 1]
```diff
@@ -25,7 +25,9 @@
 from system_settings.models import SystemSettings
 from .models import *
 from .oscal import Catalogs, Catalog
-
+from .models import *
```
Comment (bama4) [inline controls/tests.py:28]: It is best practice to import the specific models that you need instead of using `*`

Comment (Yodart) [reply controls/tests.py]: Using specific models imported on `siteapp.models`. Commited on [
f68a670](https://github.com/GovReady/govready-q/pull/1203/commits/f68a670bd2a277b0632fe3a46e5af8dbd5678b39)

File: controls/tests.py [hunk 2]
```diff
@@ -461,6 +462,27 @@ def test_element_copy(self):
...
+    def test_tests(self):
+        self.assertEqual(1,1)
+
+    def test_move_project_create(self):
+        """Test moving a project to another portfolio"""
+        initial_porfolio = Portfolio.objects.create(title="Portfolio 1") 
+        new_portfolio = Portfolio.objects.create(title="Portfolio 2") 
+        project = Project.objects.create(portfolio=initial_porfolio)
+        project.portfolio = initial_porfolio
+        self.assertTrue(initial_porfolio.id is not None)
```
Comment (bama4) [inline controls/tests.py:476]: Could use the `assertIsNotNone()` function instead

File: controls/tests.py [hunk 3]
```diff
@@ -461,6 +462,27 @@ def test_element_copy(self):
...
+        self.assertTrue(new_portfolio.id is not None)
+        self.assertTrue(project.id is not None)
+        self.assertTrue(project.portfolio.id is not None)
+        self.assertTrue(project.portfolio.title == "Portfolio 1")
```
Comment (bama4) [inline controls/tests.py:480]: Could use `assertEqual`

File: siteapp/views.py [hunk 4]
```diff
@@ -1162,6 +1162,30 @@ def rename_project(request, project):
...
+        new_portfolio = get_object_or_404(Portfolio, id=int(new_portfolio_id))
+        project.portfolio = new_portfolio
+        project.save()
+        logger.info(
+            event="move_project successful",
+            object={"project_id": project.id,"new_portfolio_id": new_portfolio.id}
+        )
+        return JsonResponse({ "status": "ok" })
+    except:
+        import sys
```
Comment (bama4) [inline siteapp/views.py:1185]: imports should go at the very top

**Comment by davidpofo** [comment]
@gregelin are we not adding proper tests for the move_project functionality here? Not sure why this was merged without these changes.

**Pull request merged**